cilium-cli/connectivity/builder/policy_local_cluster.go

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package builder
+
+import (
+	_ "embed"
+
+	"github.com/cilium/cilium/cilium-cli/connectivity/check"
+	"github.com/cilium/cilium/cilium-cli/connectivity/tests"
+	"github.com/cilium/cilium/cilium-cli/utils/features"
+)
+
+//go:embed manifests/client-egress-to-echo-no-cluster-policy.yaml
+var clientEgressToEchoNoClusterPolicyYAML string
+
+type policyLocalCluster struct{}
+
+func (t policyLocalCluster) build(ct *check.ConnectivityTest, _ map[string]string) {
+	newTest("policy-local-cluster-egress", ct).
+		WithFeatureRequirements(features.RequireEnabled(features.PolicyDefaultLocalCLuster)).
```

Comment:
Nit. Could remove the feature requirement here, and reflect this in the expectations below.